experimental/traffic-portal/src/app/core/servers/capabilities/capability-details/capability-details.component.html

```diff
@@ -0,0 +1,33 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
```

Comment:
How come the form doesn't have a `description` field? I see the column for the same in the table.

Reply:
Because in APIv4.0 Capabilities don't have `description` - so it shouldn't be in the table, either.